jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/RetrieveRegistryCredentialsStep.java

```diff
@@ -33,14 +34,17 @@
 /** Attempts to retrieve registry credentials. */
 class RetrieveRegistryCredentialsStep implements AsyncStep<Credential>, Callable<Credential> {
 
-  private static final String DESCRIPTION = "Retrieving registry credentials for %s";
+  private static String makeDescription(String registry) {
+    return "Retrieving registry credentials for " + registry;
+  }
 
   /** Retrieves credentials for the base image. */
   static RetrieveRegistryCredentialsStep forBaseImage(
       ListeningExecutorService listeningExecutorService, BuildConfiguration buildConfiguration) {
     return new RetrieveRegistryCredentialsStep(
         listeningExecutorService,
-        buildConfiguration.getBuildLogger(),
+        // TODO: Replace with ExecutionMainframe.emit so that it is never nullable.
```

Comment:
What's ExecutionMainframe?

Reply:
Ah will remove this - not needed anymore since we're using default no-op emitter.